lib/utils/fs/fse.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
This file can be removed once the `this.serverless.utils....` functionality is used.